subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/element/field/TextFieldAction.kt

```diff
@@ -57,9 +57,9 @@ internal class TextFieldAction(
                     stringToBeTyped = text
                 )
             ),
-            pressImeActionButton()
+            pressImeActionButton(),
```

Comment:
So what is changed? I didn't get the idea of that fix
Is the difference between closeSoftKeyboard or we must call pressImeActionButton differently?

Reply:
no, closeSoftKeyboard was called without waitFor